frontend/src/app/team/team.component.ts

```diff
@@ -131,14 +131,14 @@ export class TeamComponent implements OnInit {
   openPersonDialog(): void {
     const dialogRef = this.dialog.open(PersonDialogComponent, {
       width: 'fit-content',
-      data: {},
+      data: {firstName: '', lastName: ''},
     });
 
     dialogRef.afterClosed().subscribe(result => {
       if (result === null) {
         return;
       }
-      // TODO
+      // TODO: Math.random ID and you can make person without name if you remove disabled through elements.
```

Comment:
**thought:** the latter is not a problem on frontend code really. Backend will validate it eventually.